### DISCUSSION TITLE
building dockerfile locally throws error

### DISCUSSION THREAD

**Discussion by ljmerza** [mmcc-xx/WhosAtMyFeeder#21] [Q&A]
im trying to buiuld the dockerfile locally to customize my ui but i get an error ` "/model.tflite": not found` - looks like this file is missing from the repo?

```
COPY model.tflite .
```

**Comment by mmcc-xx** [comment]
I got the model from here:
https://tfhub.dev/google/lite-model/aiy/vision/classifier/birds_V1/3

I don't know if it is kosher to re-host that model or not... I guess I'm already including in the docker image so it wouldn't be any worse to put it up here.
